src/Plugins/SimplnxCore/src/SimplnxCore/Filters/ReadHDF5DatasetFilter.cpp

```diff
@@ -43,6 +44,59 @@ std::vector<size_t> createDimensionVector(const std::string& cDimsStr)
 
   return cDims;
 }
+
+template <typename T>
+Result<> fillDataStore(DataArray<T>& dataArray, const DataPath& dataArrayPath, const nx::core::HDF5::DatasetIO& datasetReader)
```

Comment:
Unused?

Reply:
That function isn't used though.

Reply:
@JDuffeyBQ Line 387 and after?

Reply:
@JDuffeyBQ I'll let you correct this section of the code then.